.github/workflows/ci-release.yml

```diff
@@ -33,7 +33,6 @@ on:
 env:
   ES_JAVA_OPTS: "-Xms256m -Xmx512m"
   BUILD_MAVEN_OPTS: "-DskipTests=true --batch-mode --also-make"
-  VERIFY_MAVEN_OPTS: "-Pcoverage"
```

Comment:
The only question. Will it stop coverage? Will we enable coverage later?

Reply:
We never had it for the tp tests.

Reply:
It will run later on the standard tests.

Reply:
Thanks!